### DISCUSSION TITLE
θέλω τη γνώμη σας για τη προσθήκη θέματος στις διαφάνειες στο συμμετοχικό περιεχόμενο

### DISCUSSION THREAD

**Discussion by p19pasc** [OompaLoompas/help#7] [Q&A]
Εφόσον στο συμμετοχικό περιεχόμενο πρέπει να κάνουμε προσθήκες με θέμα το Λογισμικό ή την κατασκευή διάδρασης, σκέφτομαι να βάλω στις διαφάνειες το Linux From Scratch. Βέβαια βλέπω ότι δεν υπάρχει κάποιο παρόμοιο θέμα και δυσκολεύομαι να το βάλω σε κάποιο συγκεκριμένο σετ διαφανειών. Εγώ πιστεύω στα [Μοντέλα](https://pibook.epidro.me/slides/models/). Καμία άποψη;

**Comment by u2nmd** [comment]
Συμφωνώ @p19pasc, πιστεύω τα Μοντέλα είναι το πιο σχετικό σετ διαφανειών για την προσθήκη της ιδέας σου!
